docs/installation/production-rack/azure.md

```diff
@@ -48,8 +48,8 @@ The following environment variables are required:
 
 ### Grant Permissions
 ```html
-    $ az ad app permission add --id $ARM_CLIENT_ID --api 00000002-0000-0000-c000-000000000000 --api-permissions 311a71cc-e848-46a1-bdf8-97ff7156d8e6=Scope 824c81eb-e3f8-4ee6-8f6d-de7f50d565b7=Role
-    $ az ad app permission grant --id $ARM_CLIENT_ID --api 00000002-0000-0000-c000-000000000000 --consent-type AllPrincipals --scope User.Read
+    $ az ad app permission add --id $ARM_CLIENT_ID --api 00000003-0000-0000-c000-000000000000 --api-permissions 1bfefb4e-e0b5-418b-a88f-73c46d2cc8e9=Role
+    $ az ad app permission grant --id $ARM_CLIENT_ID --api 00000003-0000-0000-c000-000000000000 --consent-type AllPrincipals --scope User.Read
```

Comment:
It lets the application impersonate any user but only with read access.

It would be kind of the AdminReadOnly AWS role equivalent


>     --scope              [Required] : A space-separated list of the claim values for delegated
>                                       permissions which should be included in access tokens for the
>                                       resource application (the API). For example, openid User.Read
>                                       GroupMember.Read.All. Each claim value should match the value
>                                       field of one of the delegated permissions defined by the API,
>                                       listed in the oauth2PermissionScopes property of the resource
>                                       service principal.
>     --consent-type                  : Indicates whether authorization is granted for the client
>                                       application to impersonate all users or only a specific user.
>                                       'AllPrincipals' indicates authorization to impersonate all
>                                       users. 'Principal' indicates authorization to impersonate a
>                                       specific user. Consent on behalf of all users can be granted
>                                       by an administrator. Non-admin users may be authorized to
>                                       consent on behalf of themselves in some cases, for some
>                                       delegated permissions.  Allowed values: AllPrincipals,
>                                       Principal.  Default: AllPrincipals.